### PR TITLE
MSMBuilder 3.7.0

### PR DESCRIPTION
**Pull request by brookehus**

### CODE REVIEW

File: msmbuilder/meta.yaml [hunk 1]
```diff
@@ -3,9 +3,9 @@ package:
   version: "3.6.1"
 
 source:
-    fn: msmbuilder-3.6.1.tar.gz
-    url: https://pypi.python.org/packages/75/c9/806c06d13a032e20d12c6ccbd318de783b18a99e0c49f0c497007ca2f796/msmbuilder-3.6.1.tar.gz#md5=0659b50d7f736dae2547c0ce4e74724d
-    md5: 0659b50d7f736dae2547c0ce4e74724d
+    fn: msmbuilder-3.7.0.zip
+    url : https://pypi.python.org/packages/db/7a/51e932e3b7cc19350db9b71b12aa595273050c57c7ef1e5fda9f546d192f/msmbuilder-3.7.0.zip#md5=4497f766b66e02b2088d7ee3f1864e5f
```
Comment (mpharrigan) [inline msmbuilder/meta.yaml]: extra space added here? `url :`

**Comment by brookehus** [comment]
Why are the AppVeyor builds not starting?

**Comment by brookehus** [comment]
@mpharrigan looks like appveyor requires the tar file

**Comment by mpharrigan** [comment]
how do you figure

**Comment by jchodera** [comment]
I've made some updates to `appveyor`. Can you guys rebase or merge from `master` and try again?

**Comment by brookehus** [comment]
```
Traceback (most recent call last):
  File "C:\Miniconda3\Scripts\conda-build-script.py", line 5, in <module>
    sys.exit(conda_build.cli.main_build.main())
  File "C:\Miniconda3\lib\site-packages\conda_build\cli\main_build.py", line 322, in main
    execute(sys.argv[1:])
  File "C:\Miniconda3\lib\site-packages\conda_build\cli\main_build.py", line 313, in execute
    noverify=args.no_verify)
  File "C:\Miniconda3\lib\site-packages\conda_build\api.py", line 97, in build
    need_source_download=need_source_download, config=config)
  File "C:\Miniconda3\lib\site-packages\conda_build\build.py", line 1527, in build_tree
    .format(recipe, e.pkgs) + "\n\n" + extra_help)
AttributeError: 'UnsatisfiableError' object has no attribute 'pkgs'
ERROR: File does not exist: C:\Miniconda3\conda-bld\win-32\pyemma-2.3.1-np111py34_0.tar.bz2
[conda-build-all] Error: one or more packages failed to build (win-32)
  mdtraj-1.8.0-np110py34_1.tar.bz2
  mdtraj-1.8.0-np111py34_1.tar.bz2
  pyemma-2.3.1-np110py34_0.tar.bz2
  pyemma-2.3.1-np111py34_0.tar.bz2
```

**Comment by jchodera** [comment]
We can address those `mdtraj` and `pyemma` failures in a separate PR. 

Is this ready to merge?

**Comment by brookehus** [comment]
@jchodera should be!

**Comment by brookehus** [comment]
@jchodera Since we are transitioning msmbuilder ownership, can I have merge access for conda-recipes so I can do this myself in the future? Thanks! cc: @cxhernandez

**Comment by jchodera** [comment]
> @jchodera Since we are transitioning msmbuilder ownership, can I have merge access for conda-recipes so I can do this myself in the future? Thanks! cc: @cxhernandez

Just invited you, @brookehus! Let me know if that doesn't work.

**Comment by brookehus** [comment]
@jchodera Looks like it worked! Thank you!

**Comment by brookehus** [comment]
@jchodera for the same reason, what do I need to do to get set up on the anaconda side of things, i.e. so I can change the label from rc to main? thanks!

**Comment by jchodera** [comment]
> @jchodera for the same reason, what do I need to do to get set up on the anaconda side of things, i.e. so I can change the label from rc to main? thanks!

What's your Anaconda Cloud username?

**Comment by brookehus** [comment]
just made one: brookehus

**Comment by jchodera** [comment]
Welcome to Omnia! Let me know if you have any trouble changing the label.

**Pull request merged**